src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1974,8 +2032,23 @@ internal static CommonOutputs.BinaryClassificationOutput TrainBinary(IHostEnviro
 
             return LearnerEntryPointsUtils.Train<SdcaBinaryTrainer.Options, CommonOutputs.BinaryClassificationOutput>(host, input,
                 () => new SdcaBinaryTrainer(host, input),
-                () => LearnerEntryPointsUtils.FindColumn(host, input.TrainingData.Schema, input.LabelColumn),
-                calibrator: input.Calibrator, maxCalibrationExamples: input.MaxCalibrationExamples);
+                () => LearnerEntryPointsUtils.FindColumn(host, input.TrainingData.Schema, input.LabelColumn));
+        }
+
+        [TlcModule.EntryPoint(Name = "Trainers.StochasticDualCoordinateAscentCalibratedBinaryClassifier",
```

Comment:
>StochasticDualCoordinateAscentCalibratedBinaryClassifier [](start = 47, length = 56)

I'm not altogether certain that there was any particular need to update entry-points any more than there was any need to update the command line.

Reply:
After iteration 8, entry point is not touched.

---
In reply to: [256640727](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256640727) [](ancestors = 256640727)